index.bs

```diff
@@ -476,8 +477,18 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
       <p>
         If present, the <dfn for='VideoConfiguration' dict-member>scalabilityMode</dfn>
         member represents the scalability mode as defined in [[webrtc-svc]].
+        Scalability mode is only applicable to the MediaEncodingType webrtc.
```

Comment:
Before this sentence, can you add something like: If absent, the default mode of "L1T1" (no scalability) is assumed. 

This spec hasn't done a great job w/ documenting defaults. Trying to make it better incrementally.